runtime-modules/content/src/lib.rs

```diff
@@ -235,7 +235,11 @@ decl_storage! {
 
         /// Global weekly NFT limit.
         pub GlobalWeeklyNftLimit get(fn global_weekly_nft_limit):
-            LimitPerPeriod<T::BlockNumber>;
+        LimitPerPeriod<T::BlockNumber>;
```

Comment:
The formatting is now inconsistent:
```suggestion
            LimitPerPeriod<T::BlockNumber>;
```